packages/video_player/video_player_windows/pigeons/messages.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/src/messages.g.dart',
+  dartTestOut: 'test/test_api.g.dart',
+  cppHeaderOut: 'windows/messages.h',
+  cppSourceOut: 'windows/messages.cpp',
+  cppOptions: CppOptions(
+    namespace: 'Messages',
+  ),
+  copyrightHeader: 'pigeons/copyright.txt',
+))
+class TextureMessage {
```

Comment:
It's not critical, but since this is a new implementation I would rather not carry this legacy style over. `video_player` was the first real testbed for Pigeon, and this approach of having a message object for everything predates Pigeon's ability to have multiple and non-class arguments. It should be trivial to just inline all of the fields as arguments now, and not need the classes.

Reply:
Oh yeah, absolutely! This simplified the pigeon file a LOT.